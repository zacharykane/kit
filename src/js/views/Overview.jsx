import React from 'react';
import PropTypes from 'prop-types';
import { css } from '@emotion/core';

import { headerTextStyles } from '../styles';

const Overview = props => {
    return (
        <section
            css={css`
                position: absolute;
                top: 0;
                right: 0;
                bottom: 0;
                left: 0;
            `}
        >
            <h1 css={headerTextStyles}>Hello {props.name}</h1>
            <p>
                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Hic
                nemo fugit quisquam tempora praesentium voluptatum cumque quod
                illo, vero reiciendis earum repudiandae quaerat quo similique
                ipsa omnis id suscipit magni?
            </p>
        </section>
    );
};

Overview.propTypes = {
    name: PropTypes.string.isRequired,
};

export default Overview;
