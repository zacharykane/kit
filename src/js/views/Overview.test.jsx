import React from 'react';
import { mount } from 'enzyme';

import Overview from './Overview';

test('Overview', () => {
    const output = mount(<Overview />);

    expect(output).toMatchSnapshot();
});
