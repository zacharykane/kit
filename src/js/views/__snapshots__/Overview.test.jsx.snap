// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview 1`] = `
"<ForwardRef(render) css={{...}} __EMOTION_TYPE_PLEASE_DO_NOT_USE__=\\"section\\" __EMOTION_LABEL_PLEASE_DO_NOT_USE__=\\"Overview\\">
  <ForwardRef(render) css={{...}} __EMOTION_TYPE_PLEASE_DO_NOT_USE__=\\"h1\\" __EMOTION_LABEL_PLEASE_DO_NOT_USE__=\\"Overview\\">
    Hello 
  </ForwardRef(render)>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Hic nemo fugit quisquam tempora praesentium voluptatum cumque quod illo, vero reiciendis earum repudiandae quaerat quo similique ipsa omnis id suscipit magni?
  </p>
</ForwardRef(render)>"
`;
